util/printer/wide_printer.go

```diff
@@ -0,0 +1,121 @@
+// Copyright © 2018 Infostellar, Inc.
```

Comment:
Did you copy any code?

Reply:
No, I got the idea to use tabwriter from kubernetes code, but don't copy from it.

Reply:
Comments of tabwriter parameters are from Go documents.